app/models/miq_vim_broker_worker/runner.rb

```diff
@@ -56,8 +56,8 @@ def enable_broker_update_notification
 
     _log.info("#{log_prefix} Enabling Broker's Update Manager Notification")
 
-    @exclude_props = VMDB::Config.new("broker_notify_properties").config[:exclude] || {}
-    @exclude_props.deep_stringify_keys!
+    @exclude_props =
+      Settings.broker_notify_properties.exclude.map { |k, v| [k.to_s, v.to_set] }.to_h
```

Comment:
The confusion is possibly there and we can always change the ones that are weird.  Also see https://github.com/railsconfig/config/blob/master/lib/config/options.rb#L109-L110